python/tk_framework_desktopserver/shotgun/scripts/execute_command.py

```diff
@@ -19,7 +19,7 @@ def execute(config, project, name, entities, base_configuration, engine_name):
 
     # Setup the bootstrap manager.
     toolkit_mgr = sgtk.bootstrap.ToolkitManager()
-    toolkit_mgr.plugin_id = "shotgun_toolkit_command_execution"
+    toolkit_mgr.plugin_id = "basic.shotgun.execute"
```

Comment:
Does the semaphone that you have prevent two bootstraps to happen at the same time? If that is the case, you won't have two bootstrapping happening at the same time. I think.

Ideas ? :)